docs/FAQ/Static-Token-Deprecation-FAQ.md

```diff
@@ -32,4 +32,8 @@ Please see [here](https://kanopy.corp.mongodb.com/docs/corpsecure/auth_flow/#ref
 
 ## Why am I being asked to copy and paste a link instead of the link being opened in a browser automatically?
 
-We implemented it this way so that it is compatible with spawn hosts. 
\ No newline at end of file
+We implemented it this way so that it is compatible with spawn hosts. 
+
+
```

Comment:
nit: extra new line here that should be moved down below the new header to separate it from it's description